test/celeritas/CMakeLists.txt

```diff
@@ -434,11 +434,6 @@ if(CELERITAS_USE_Geant4 AND CELERITAS_USE_ROOT)
     add_custom_command(
       TARGET update-root-test-data
       POST_BUILD
-      DEPENDS
```

Comment:
Consider adding to the commit logs that `DEPENDS` is not used in case of `Build Event/POST_BUILD add_custom_command` to point out the reason for the invalidity.